app/models/offline_transaction_charge.rb

```diff
@@ -7,6 +7,7 @@
 class OfflineTransactionCharge < ApplicationRecord
 	include Model::SubtransactionPaymentable
 	has_one :legacy_payment, class_name: 'Payment', through: :subtransaction_payment
+	has_one :offsite_payment
```

Comment:
For simplicity, we can go through `legacy_payment` to get to the original OffsitePayment. This avoids needing to change the db schema. 

This should work:

```suggestion
	has_one :offsite_payment, through: :legacy_payment
```